dev-python/python-msrplib/python-msrplib-0.19.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="Client library for MSRP protocol and its relay extension"
+HOMEPAGE="http://sipsimpleclient.org"
+SRC_URI="https://github.com/AGProjects/${PN}/archive/release-${PV}.tar.gz"
+S=${WORKDIR}/${PN}-release-${PV}
+
+LICENSE="LGPL-2+"
+SLOT="0"
+IUSE="test"
+
+KEYWORDS="~amd64 ~x86"
+#RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/python-application[${PYTHON_USEDEP}]
+	dev-python/python-eventlib[${PYTHON_USEDEP}]
+	dev-python/python-gnutls[${PYTHON_USEDEP}]
+	dev-python/twisted[${PYTHON_USEDEP}]
+"
+
+DEPEND="test? ( dev-python/greenlet[${PYTHON_USEDEP}] )"
+
+python_test() {
+        pushd test > /dev/null || die
+        "${EPYTHON}" -m unittest discover -v || die
+        popd > /dev/null || die
+}
+
+src_compile() {
```

Comment:
Why are you doing this? If you override the whole build & install process like this, I'm not surprised it can't find anything.

Reply:
it may be that time ago I felt there was need to rewrite it for any reason and it went there,
it doesn't seem to be needed anymore.